app/src/main/res/values-night/colors.xml

```diff
@@ -21,7 +21,7 @@
 
 <resources>
 
-    <color name="background">#34373D</color>
+    <color name="background">#17181A</color>
```

Comment:
Just checking that it's not a leftover ?

Reply:
I noticed that the background color for night mode is lighter than on designs and it turned out in this xml it's different than the default one in our theme and designs so I updated that